packages/client/src/admin/share/RelationTable.js

```diff
@@ -0,0 +1,135 @@
+import React, { PureComponent } from 'react';
+import { Icon, Table, Button } from 'antd';
+import { remove } from 'lodash';
```

Comment:
```suggestion
import { remove, difference } from 'lodash';
```

and remove below `difference` import